COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
@peti I'm not sure why people didn't add their names either. If you want, I can extract the names of all committers from the git log and add them here so there won't be false exclusivity. How does that sound?